tasks/full_execution.yml

```diff
@@ -35,8 +35,7 @@
         ovirt_engine_setup_firewall_manager: null
         ovirt_engine_setup_answer_file_path: /root/ovirt-engine-answers
         ovirt_engine_setup_use_remote_answer_file: true
-        ovirt_engine_setup_update_all_packages: false
-        ovirt_engine_setup_offline: true
+        ovirt_engine_setup_offline: "{{ he_offline_deployment | default(false) }}"
```

Comment:
I think it will be better to add the default value to defaults/main.yml file instead of here